package.json

```diff
@@ -17,7 +18,8 @@
     "check": "npm run -s lint && npm run flow",
     "lint": "eslint './src/**/*.js' './test/**/*.js'",
     "flow": "flow check",
-    "bundle-size": "webpack-bundle-analyzer"
+    "bundle-size": "webpack-bundle-analyzer",
+    "test": "jest"
```

Comment:
As discussed, I think we should run the test on prepublish to make CI fail if unit tests are failing

Reply:
Added 👍